shell/browser/notifications/win/notification_presenter_win.cc

```diff
@@ -64,8 +64,10 @@ bool NotificationPresenterWin::Init() {
 std::wstring NotificationPresenterWin::SaveIconToFilesystem(
     const SkBitmap& icon,
     const GURL& origin) {
-  std::string filename;
+  if (icon.drawsNothing() || icon.width() <= 0 || icon.height() <= 0)
```

Comment:
Doesn't [drawsNothing](https://source.chromium.org/chromium/chromium/src/+/main:third_party/skia/include/core/SkSize.h;l=30-31) cover the other two conditions ?